files-override/tests/index.html

```diff
@@ -28,8 +28,14 @@
     <script src="/@embroider/core/vendor.js"></script>
     <script src="/@embroider/core/test-support.js"></script>
     <script src="/@embroider/core/entrypoint" type="module"></script>
-    <script src="/@embroider/core/test-entrypoint" type="module"></script>
+    <script type="module">import "ember-testing";</script>
 
-    {{content-for "body-footer"}} {{content-for "test-body-footer"}}
+    <script type="module">
+      import { start } from './test-helper';
+      import.meta.glob("./**/*.{js,gjs,gts}", { eager: true });
+      start();
+    </script>
+
+    {{content-for "body-footer"}}
```

Comment:
what about test-body-footer?

Reply:
it was removed in the upstream PR so I removed it here 👍 https://github.com/embroider-build/embroider/pull/2071/files#diff-1737613795478ebf91b6ce343eca8e5044e9c4e07eef6259f463ce3e3b5f550fL32